src/HttSystematics_MSSMvsSMRun2.cc

```diff
@@ -1442,6 +1905,11 @@ void AddMSSMvsSMRun2Systematics(CombineHarvester &cb, bool jetfakes, bool embedd
             .process({"jetFakes"})
             .AddSyst(cb, "CMS_ff_total_qcd_stat_iso_"+unc+"_$CHANNEL_$ERA", "shape", SystMap<>::init(1.00));
 
+        cb.cp()
+            .channel({"et", "mt"})
+            .process({"jetFakes"})
+            .AddSyst(cb, "CMS_ff_total_qcd_stat_os_"+unc+"_$CHANNEL_$ERA", "shape", SystMap<>::init(1.00));
+
```

Comment:
For this uncertainty we will need to reproduce the friend trees and the shapes in order to have them ready. We hadn't included them so far because they were not listed on the twiki.

Reply:
Yes, of course. Do you have a rough estimate on the timescale of the updates? I am asking to better gauge if it makes sense to start the new friend trees now or if it would make sense to wait for the further updates

Reply:
A couple of days. We have a few ideas and are trying to implement them to see how much the FF systematics would increase by.